package/src/specs/RNCandle.nitro.ts

```diff
@@ -357,9 +410,12 @@ export interface RNCandle extends HybridObject<{ ios: "swift" }> {
   ): void;
   initialize(appUser: AppUser): void;
   getLinkedAccounts(): Promise<LinkedAccount[]>;
-  unlinkAccount(linkedAccountID: string): Promise<void>;
+  getLinkedAccount(ref: LinkedAccountRef): Promise<LinkedAccount>;
+  unlinkAccount(ref: DeleteLinkedAccountRef): Promise<void>;
```

Comment:
```suggestion
  unlinkAccount(ref: LinkedAccountRef): Promise<void>;
```
#req duplicate model